source/PlayerInfo.cpp

```diff
@@ -3138,7 +3139,12 @@ void PlayerInfo::Save(const string &path) const
 		out.Write("plugins");
 		out.BeginChild();
 		for(const auto &plugin : GameData::PluginAboutText())
-			out.Write(plugin.first);
+		{
+			if(Plugins::IsEnabled(plugin.first))
+				out.Write(plugin.first);
```

Comment:
I think this should include the state.
```suggestion
				out.Write(plugin.first, "(enabled)");
```

Reply:
I can add this.  I implied if it didn't have disabled that it was enabled but we can make it more explicit.

Reply:
I added this and attributed you.